test/integration/roles/test_ec2_eip/tasks/main.yml

```diff
@@ -152,7 +152,7 @@
 - name: assert new EIP was assigned
   assert:
     that:
-      - '"failed" not in result'
```

Comment:
Ok, but a lot of tasks simply checked that 'failed' was not returned, which we now have modified.
I will remove all of them ?